HW02/Mikhail_Sauchuk/mikhail_sauchuk_hw_02_t_01.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Method `task_1` should return:
+# - full text of the first line with an error
+# (the presence of the substring `error` in any case)
+# - If the input is text that does not contain the word `error`,
+# return an empty string
+
+LOG_FILE = 'bash.log'
+
+def find_error(file_name)
+  # method rerturns first line with error from the input log
+  File.open(file_name, 'r') do |f|
```

Comment:
You can use one line syntax 
```suggestion
  File.readlines(file_name)
```

Reply:
I have refactored three tasks with your suggestions, thanks